dataRetriever/interface.go

```diff
@@ -83,6 +83,8 @@ const (
 	MiniblockHashByTxHashUnit UnitType = 14
 	// ReceiptsUnit is the receipts storage unit identifier
 	ReceiptsUnit UnitType = 15
+	// SmartContractUnit is the smart contracts storage unit identifier
+	SmartContractUnit UnitType = 16
```

Comment:
please adapt the comment on line 82 as to match the function's name

Reply:
done